# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/patches/default-system-fpicker.diff
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- officecfg/registry/data/org/openoffice/Office/Common.xcu
+++ officecfg/registry/data/org/openoffice/Office/Common.xcu
@@ -554,8 +554,8 @@
     </node>
   </node>
   <node oor:name="Misc">
-    <prop oor:name="UseSystemFileDialog"  install:module="UseOOoFileDialogs">
-      <value>false</value>
+    <prop oor:name="UseSystemFileDialog"  install:module="UseOOoFileDialogs"> <!-- safe -->
+      <value>true</value>
     </prop>
   </node>
   <node oor:name="Save">
